torch/csrc/jit/pybind_utils.h

```diff
@@ -328,7 +325,8 @@ inline IValue toIValue(
     case TypeKind::CompleteTensorType: {
       auto var = py::cast<autograd::Variable>(obj);
       if (var.is_sparse()) {
-        AT_ERROR("sparse tensors not supported");
+        AT_WARN(
+            "Sparse tensors in TorchScript is experimental and likely to break");
```

Comment:
-> "Using sparse tensors in TorchScript is experimental. Many optimization pathways have not been thoroughly tested with sparse tensors. Please include the fact that the network is running sparse tensors in any bug reports submitted."